src/components/table/table--sortable.njk

```diff
@@ -0,0 +1,227 @@
+
+<table class="usa-table">
+  <caption>Recently admitted US states (sortable table example)</caption>
+  <thead>
+    <tr>
+      <th data-sortable scope="col" role="columnheader">
+        Name
+      </th>
+      <th data-sortable scope="col" role="columnheader" aria-sort="descending">
+        Order admitted to union
+      </th>
+      <th data-sortable scope="col" role="columnheader">
+        Date of ratification vote
+      </th>
+      <th data-sortable scope="col" role="columnheader">
+        Date admitted to union
+      </th>
+      <th data-sortable scope="col" role="columnheader">
+        Percent of voters in favor of ratification 
+      </th>
+      <th data-sortable scope="col" role="columnheader">
+        Votes cast in favor of ratification
+      </th>
+      <th data-sortable scope="col" role="columnheader">
+        Estimated population at time of admission
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th scope="row" role="rowheader">Hawaii</th>
+      <td data-sort-value="50">50th</td>
+      <td data-sort-value="331844400">Jun. 27, 1959</td>
```

Comment:
where are these sort values coming from?

Reply:
this number is a Unix timestamp

Reply:
Would it be worth in the guidance adding indicating supporting additional formats like YYYY:MM:DD? Thinking of support for tools where Unix timestamps might be as readily available.

Reply:
I've deferred parsing dates from strings because they're finicky and would probably have weird issues across timezones unless we included a library like Moment.JS or one of its successors (which I am loathe to do).